scripts/scil_compute_fodf_metrics.py

```diff
@@ -47,14 +46,6 @@ def _build_arg_parser():
     p.add_argument(
         'input', metavar='fODFs',
```

Comment:
in_fODFs

Reply:
Is it right?

Reply:
I would say `in_fODF` as in `scil_compute_mean_fixel_afd_from_bundles.py`
But I would prefer `in_fodf` in both

Reply:
I put in_fODF and out_fODF everywhere in my PR.